templates/components/codeblock_templ.go

```diff
@@ -0,0 +1,54 @@
+// Code generated by templ@v0.2.408 DO NOT EDIT.
+
+package components
+
+//lint:file-ignore SA4006 This context is only used if a nested component is present.
+
+import "github.com/a-h/templ"
+import "context"
+import "io"
+import "bytes"
+
+import (
+	"gopkg.in/yaml.v3"
+)
+
+func CodeBlock(value any) templ.Component {
```

Comment:
any

Reply:
This any is not templ related. It's just how I happened to implement this one. I moved the "yaml marshalling" into this function, so I thought `any` was appropriate. I can certainly change it.

Suggest I do?